source/portable/NetworkInterface/ksz8851snl/NetworkInterface.c

```diff
@@ -939,11 +941,11 @@ static void ksz8851snl_update()
                int rxHead = xMicrelDevice.us_rx_head;
                /* RX step18-19: pad with dummy data to keep dword alignment. */
                /* Packet lengths will be rounded up to a multiple of "sizeof size_t". */
-/*			xLength = xMicrelDevice.rx_buffers[ rxHead ]->xDataLength & 3; */
-/*			if( xLength != 0 ) */
-/*			{ */
-/*				ksz8851_fifo_dummy( 4 - xLength ); */
-/*			} */
+               /* xLength = xMicrelDevice.rx_buffers[ rxHead ]->xDataLength & 3; */
```

Comment:
Why do we have /* */ in every line, it should be one covering all as mentioned in style guideline https://www.freertos.org/FreeRTOS-Coding-Standard-and-Style-Guide.html

Reply:
Updated